### PR TITLE
test: create bq table issue is fixed

### PR DESCRIPTION
**Pull request by t-karasova**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-retail/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: samples/interactive-tutorials/product/add_fulfillment_places.py [hunk 1]
```diff
@@ -64,7 +62,7 @@ def add_fulfillment_places(product_name: str, timestamp, place_id):
     # This is a long running operation and its result is not immediately present with get operations,
     # thus we simulate wait with sleep method.
     print("---add fulfillment places, wait 40 seconds :---")
```
Comment (kweinmeister) [inline samples/interactive-tutorials/product/add_fulfillment_places.py:64]: Update comment to match (now 70 seconds)

Comment (kweinmeister) [reply samples/interactive-tutorials/product/add_fulfillment_places.py:64]: Also, why 70 seconds? Why not pick 60 or 90 seconds that would look more standard?

Comment (chenlei1216) [reply samples/interactive-tutorials/product/add_fulfillment_places.py:64]: +1 for 90 seconds to be more stable for testing, and fix the doc above. Also should we use the same wait tie for remove/set APIs as well?

**Review by chenlei1216 — APPROVED** [review]

**Review by kweinmeister — APPROVED** [review]

File: samples/interactive-tutorials/product/add_fulfillment_places.py (hunk 1 above)
Comment (t-karasova) [reply samples/interactive-tutorials/product/add_fulfillment_places.py:64]: increased to 90

**Review by parthea — APPROVED** [review]

**Pull request merged**